package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "csharp",
   "publisher": "ms-vscode",
-  "version": "1.1.7",
+  "version": "1.1.8",
```

Comment:
I think we should go to 1.2.0 since:
1. This is a breaking change (we are loosing rc2 debugging)
2. Leaves us space to release more 1.1.x vsix's (ex: to fix OmniSharp)